requirements.txt

```diff
@@ -1,2 +1,3 @@
 diff_match_patch
-# additionally https://github.com/NVIDIA/NeMo is required
\ No newline at end of file
+nemo_toolkit[all]
```

Comment:
Also any reason why we need to install "all"? It takes quite a bit longer, I think and we don't really use everything from nemo. In fact, I'd prefer if we make nemo installation optional as we really only need it for inference on models, which is not always running. We currently also import it to use logging, but that's really an overkill and we'd be better off by just using logging from python direclty